components/DashboardPage/components/Pomodoro/Timer.tsx

```diff
@@ -0,0 +1,103 @@
+import moment from 'moment';
+import { useState, useEffect } from 'react';
+import { Stack, Typography, Button } from 'components';
+
+interface TimerProps {
+  round: number;
+  setRound: any;
+}
+
+export function Timer({ round, setRound }: TimerProps) {
+  let workTime = 25 * 60;
+  let breakTime = 5 * 60;
+  const [time, setTime] = useState<number>(workTime);
+  const [clock, setClock] = useState<boolean>(true);
+  const [countdown, setCountdown] =
+    useState<boolean>(false);
+  const [workState, setWorkState] =
+    useState<boolean>(false);
+
+  useEffect(() => {
+    if (time === 0) {
+      setWorkState((workState) => !workState);
+      if (workState === false) {
+        setTime(breakTime);
+        setRound((round: number) => round + 1);
+      }
+      if (workState === true) setTime(workTime);
+    }
+
+    if (countdown && time > 0) {
+      setTimeout(() => {
+        setClock((clock) => !clock);
+      }, 1000);
+      setTime(time - 1);
+    }
+  }, [countdown, clock, workState]);
+
+  var momentDurationFormatSetup = require('moment-duration-format');
+
+  const formattedTimeLeft = moment
+    .duration(time, 's')
+    .format('mm:ss', { trim: false });
```

Comment:
Typescript is complaining for no reason here on my end. No big deal, just thought of letting you know in case you don't have that ts error.

Reply:
same! not sure why

Reply:
What's the error? 🤔 👀

Reply:
It was not an error that breaks the code compilation. Maybe it was just on my computer.

Reply:
I think the build is failing because of this error, ion know how to fix it thou 👀🥴